db/data/regions/IN/en.yml

```diff
@@ -8,17 +8,26 @@ en:
             default: State
             optional: State (optional)
           errors:
-            blank: Select a state
+            blank_instructional: Select a state
+            blank_informative: State is missing
         zip:
           label:
             default: PIN code
             optional: PIN code (optional)
           errors:
-            blank: Enter a PIN code
-            invalid_for_country: Enter a valid PIN code for %{country}
-            invalid_for_country_and_province: Enter a valid PIN code for %{province},
+            blank_instructional: Enter a PIN code
+            blank_informative: PIN code is missing
+            invalid_for_country_instructional: Enter a valid PIN code for %{country}
+            invalid_for_country_informative: The PIN code doesn't match %{country}
```

Comment:
We ask for a valid PIN code.. but then say it doesn't match.. should we say `The PIN code isn't valid for %{country}`?